src/mscorlib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -537,10 +544,8 @@ internal Container Resize(int newSize)
                         if (primary != null)
```

Comment:
If the `primary` is `null`, who is responsible for deallocating this Handle?  Do you need to set the `hashCode` to -1, to indicate that ownership has not been transferred to the newer `Container`?